include/commissioner/defines.hpp

```diff
@@ -123,6 +123,11 @@ static constexpr uint16_t kDefaultAeUdpPort = 1001;
  */
 static constexpr uint16_t kDefaultNmkpUdpPort = 1002;
 
+/**
+ * The default management UDP Port used by TMF transaction Thread devices if not specified.
```

Comment:
Do we allow specifying the port? It looks like it's hard coded in the implementation.

Reply:
Currently no, my plan is to support all/any port which the request is targeting, but it would require more changes to the coap library. Let this be a short-term workaround to unblock the diag command development. Thoughts?

Reply:
In the OT implementation, the sockport is also specified to default management port.

https://github.com/openthread/openthread/blob/main/src/core/thread/tmf.hpp#L91

Reply:
My concern is on the comment. It looks like this port can be specified.

Reply:
Updated Thanks @bukepo's comment.